content/docs/guides/install.md

```diff
@@ -53,7 +53,7 @@ You can configure the OSM installation by overriding the values file.
 
 #### Helm install
 
-Then run the following `helm install` command. The chart version can be found in the Helm chart you wish to install [here](https://github.com/openservicemesh/osm/blob/release-v0.9/charts/osm/Chart.yaml#L17).
+Then run the following `helm install` command. The chart version can be found in the Helm chart you wish to install [here](https://github.com/openservicemesh/osm/blob/release-{{< param osm_version >}}/charts/osm/Chart.yaml#L17).
```

Comment:
If we're going to template the version, I'd vote to omit any anchors to specific lines (`#L17`). I also noticed links to line numbers in content/docs/guides/certificates.md.

Actually, I'm not sure that sentence containing the link is really necessary at all. I think it makes more sense to just pass a particular version to the `helm install` command below if the docs correspond to that particular version, though I know that would be pushing the scope for this PR.

Reply:
I agree with removing the anchors to specific lines. I have been removing them when creating the release-specific sites, but have not made the change on main. I can remove them in a follow-up PR.